### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by karenkrieger**
#### What problem is this solving?

Updating the Product Recommendation documentation

#### How to test it?

[Workspace](Link goes here!)

### CODE REVIEW

File: docs/README.md [hunk 1]
```diff
@@ -92,106 +107,177 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
+  "flex-layout.row#recommendation-shelf": {
+    "children": ["recommendation-shelf"]
+  },
+  "recommendation-shelf":{
+      "props": {
+           "strategy": MOST_POPULAR,
+           "secondaryStrategy": NEW_RELEASES,
+           "recommendation":{
+ 
+           }
```
Comment (klzns) [inline docs/README.md:156]:
```suggestion
           "strategy": "MOST_POPULAR",
           "secondaryStrategy": "NEW_RELEASES",
           "recommendation": {}
```

File: docs/README.md [hunk 2]
```diff
@@ -92,106 +107,177 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
+| `BEST_CHOICE`             | Fetches and displays the most visited and ordered products based on the similarity of products on the currently displayed product page. This prop only works in the theme template `store.product` and `store.search`. | 
+| `BOUGHT_TOGETHER`         | Fetches and displays recommendations for products often bought together based on the similarity of products on the currently displayed product page. This prop only works in the theme template `store.product` and `store.search`. | 
+| `RECOMMENDATION_HISTORY` | Fetches and displays products based on the user's navigation history.   |
+| `CART_HISTORY`           | Fetches and displays products based on the user's cart history.         | 
+| `ORDER_HISTORY`          | Fetches and displays products based on the user's order history.        |
+ 
+See the example below to know how to add it to your template:
+ 
+```json
+"store.home": {
```
Comment (klzns) [inline docs/README.md:142]:
```suggestion
{
  "store.home": {
```

File: docs/README.md [hunk 3]
```diff
@@ -92,106 +107,177 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
+  "recommendation-shelf":{
+      "props": {
+           "strategy": MOST_POPULAR,
+           "secondaryStrategy": NEW_RELEASES,
+           "recommendation":{
+              "count":{
+                "minimum": 5,
+                "recommendations": 20
+             }
+
```
Comment (klzns) [inline docs/README.md:208]:
```suggestion
```

File: docs/README.md [hunk 4]
```diff
@@ -92,106 +107,177 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
+      "flex-layout.row#recommendation-shelf",
+    ]
+  },
+  "flex-layout.row#recommendation-shelf": {
+    "children": ["recommendation-shelf"]
+  },
+  "recommendation-shelf":{
+      "props": {
+           "strategy": MOST_POPULAR,
+           "secondaryStrategy": NEW_RELEASES,
```
Comment (klzns) [inline docs/README.md:202]:
```suggestion
           "strategy": "MOST_POPULAR",
           "secondaryStrategy": "NEW_RELEASES",
```

File: docs/README.md [hunk 5]
```diff
@@ -92,106 +107,177 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
 | Prop name         | Type      | Description                                                    | Default value |
 | ----------------- | --------- | -------------------------------------------------------------- | ------------- |
 | `minimum`         | `number`  | Defines the minimum recommendations that should be fetched.   | `5`             |
 | `recommendations` | `number`  | Defines the total number of recommendations that should be fetched. | `20`           |
-
+ 
+See the example below to know how to add it to your template:
+ 
+```json
+"store.home": {
```
Comment (klzns) [inline docs/README.md:191]:
```suggestion
{
  "store.home": {
```

**Review by klzns — CHANGES_REQUESTED** [review]

**Review by brunoamui — APPROVED** [review]
LGTM

File: docs/README.md [hunk 6]
```diff
@@ -92,106 +107,176 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
-
+ 
 | Prop name            | Type      | Description                                                                      | Default value      |
 | -------------------- | --------- | -------------------------------------------------------------------------------- | ------------------ |
-| `strategy`           | `enum`    | Strategy that will be used to fetch and display the recommended products. Possible values can be find in the table below.   | `RECOMMENDATION_HISTORY` |
-| `secondaryStrategy`  | `enum`    | Secondary strategy that will be used to fetch and display the recommended products if the initial strategy does not return results. Possible values can be find in the table below. | `RECOMMENDATION_HISTORY` |
+| `strategy`           | `enum`    | Strategy used to fetch and display the recommended products. Possible values can be found in the [table](link pra tabela de estratégias).   | `RECOMMENDATION_HISTORY` |
+| `secondaryStrategy`  | `enum`    | Secondary strategy used to fetch and display the recommended products if the initial strategy does not return results. Possible values can be found in the [table](link pra tabela de estratégias). | `RECOMMENDATION_HISTORY` |
```
Comment (thalytafabrine) [inline docs/README.md:247]: same. remembering that the possible strategies for the `recommendation-refresh` are not the same as the `recommendation-shelf`, so we need to create the table with the possible strategies for it.

Comment (thalytafabrine) [reply docs/README.md]: it's easier to say the strategies that apply to it hahaha 
the only ones that work with this shelf are: `RECOMMENDATION_HISTORY`,` ORDER_HISTORY` and `CART_HISTORY`

Comment (karenkrieger) [reply docs/README.md]: Done! ✔️

**Review by thalytafabrine — APPROVED** [review]

File: docs/README.md [hunk 7]
```diff
@@ -92,106 +107,182 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
+    "children": ["recommendation-shelf"]
+  },
+  "recommendation-shelf":{
+      "props": {
+           "strategy": "MOST_POPULAR",
+           "secondaryStrategy": "NEW_RELEASES",
+           "recommendation": {}
+       }
+  }
+}
```
Comment (klzns) [inline docs/README.md:158]:
```suggestion
{
  "store.home": {
    "blocks": ["flex-layout.row#recommendation-shelf"]
  },
  "flex-layout.row#recommendation-shelf": {
    "children": ["recommendation-shelf"]
  },
  "recommendation-shelf": {
    "props": {
      "strategy": "MOST_POPULAR",
      "secondaryStrategy": "NEW_RELEASES",
      "recommendation": {}
    }
  }
}
```

File: docs/README.md [hunk 8]
```diff
@@ -92,106 +107,182 @@ Now, you are able to use all blocks exported by the `recommendation-shelf` app.
...
+           "secondaryStrategy": "NEW_RELEASES",
+           "recommendation":{
+              "count":{
+                "minimum": 5,
+                "recommendations": 20
+             }
+           }
+       }
+  }
+}
```
Comment (klzns) [inline docs/README.md:211]:
```suggestion
{
  "store.home": {
    "blocks": ["flex-layout.row#recommendation-shelf"]
  },
  "flex-layout.row#recommendation-shelf": {
    "children": ["recommendation-shelf"]
  },
  "recommendation-shelf": {
    "props": {
      "strategy": "MOST_POPULAR",
      "secondaryStrategy": "NEW_RELEASES",
      "recommendation": {
        "count": {
          "minimum": 5,
          "recommendations": 20
        }
      }
    }
  }
}
```

**Review by klzns — APPROVED** [review]

**Pull request merged**